letsdrill.rb

```diff
@@ -1,13 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+case integer
+when 80..89
+  return "B"
+when 70..79
+  return "C"
+when 60..69
+  return "D"
+else
+  if integer > 90
+    return "A"
+  else
+  return "F"
+end
+end
```

Comment:
You could totes make this a little more readable. Maybe try putting the range for an "A" at the top and make "F" a default case.